lib/rubocop/cop/lint/shadowed_exception.rb

```diff
@@ -30,6 +28,22 @@ module Lint
       #   rescue Exception
       #     handle_exception
       #   end
+      #
+      #   # good, however depending on runtime environment.
+      #   #
+      #   # This is a special case for system call errors.
+      #   # System dependent error code depends on runtime environment.
+      #   # For example, whether `Errno::EAGAIN` and `Errno::EWOULDBLOCK` are
+      #   # the same error code or different error code depends on environment.
+      #   # This good case is for `Errno::EAGAIN` and `Errno::EWOULDBLOCK` with
+      #   # the same error code.
+      #   # It is a bad case in different error code environment.
```

Comment:
It shouldn't be a bad case in any environment.

Reply:
Oops! This is my mistake. Thanks.